File: fix.py
import os
import glob
import re

directories = ['src/components', 'src/pages']

def fix_colors(content):
    content = content.replace('bg-forest-dark', 'bg-[#0A1E11]')
    content = content.replace('text-forest-dark', 'text-[#0A1E11]')
    content = content.replace('border-forest-dark', 'border-[#0A1E11]')
    content = content.replace('bg-forest-green', 'bg-[#113F2B]')
    content = content.replace('text-forest-green', 'text-[#113F2B]')
    content = content.replace('text-accent-orange', 'text-[#E65C00]')
    content = content.replace('bg-accent-orange', 'bg-[#E65C00]')
    content = content.replace('border-accent-orange', 'border-[#E65C00]')
    content = content.replace('text-accent-green', 'text-[#34D399]')
    content = content.replace('bg-accent-green', 'bg-[#34D399]')
    
    # Fix opacity modifiers if needed, but since we use arbitrary values like text-[#0A1E11]/80,
    # Tailwind 4 fully supports them! Hex with slash opacity parses perfectly: text-[#0A1E11]/80.
    
    # Fix Hero Video path
    content = content.replace('/Hero Video.mp4', '/hero_video.mp4')
    
    return content

for d in directories:
    for filepath in glob.glob(os.path.join(d, '*.tsx')):
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        new_content = fix_colors(content)
        
        if content != new_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(new_content)
            print("Updated", filepath)

File: fix.mjs
import fs from 'fs';
import path from 'path';

const dirs = ['src/components', 'src/pages'];

function fixColors(content) {
    let newContent = content.replace(/bg-forest-dark/g, 'bg-[#0A1E11]');
    newContent = newContent.replace(/text-forest-dark/g, 'text-[#0A1E11]');
    newContent = newContent.replace(/border-forest-dark/g, 'border-[#0A1E11]');
    newContent = newContent.replace(/bg-forest-green/g, 'bg-[#113F2B]');
    newContent = newContent.replace(/text-forest-green/g, 'text-[#113F2B]');
    newContent = newContent.replace(/text-accent-orange/g, 'text-[#E65C00]');
    newContent = newContent.replace(/bg-accent-orange/g, 'bg-[#E65C00]');
    newContent = newContent.replace(/border-accent-orange/g, 'border-[#E65C00]');
    newContent = newContent.replace(/text-accent-green/g, 'text-[#34D399]');
    newContent = newContent.replace(/bg-accent-green/g, 'bg-[#34D399]');

    // Fix Hero Video.mp4 path
    newContent = newContent.replace(/\/Hero Video\.mp4/g, '/hero_video.mp4');
    return newContent;
}

dirs.forEach(d => {
    const fullPath = path.resolve(process.cwd(), d);
    fs.readdirSync(fullPath).forEach(file => {
        if (file.endsWith('.tsx')) {
            const filepath = path.join(fullPath, file);
            const content = fs.readFileSync(filepath, 'utf-8');
            const newContent = fixColors(content);
            if (content !== newContent) {
                fs.writeFileSync(filepath, newContent, 'utf-8');
                console.log('Updated ' + filepath);
            }
        }
    });
});
